### PR TITLE
Fix #1074, add extra info re pa, make sure ppl choose a free account

### LINKED ISSUE #1074: More info around pythonanywhere
A couple of attendees were a little confused by pythonanywhere.
Firstly they were a little surprised at having to enter credit card details so perhaps a warning about that in the tutorial.
Secondly, people thought they maybe had to create the account on pythonanywhere to be able to use python, so maybe a short sentence or two to differentiate python and pythonanywhere.

### PR DESCRIPTION
**Pull request by hjwp**
adds some brief explanatory text re: what pythonanywhere is, and an extra screenshot (and text) to make sure people choose the free/beginner account (and they don't need a credit card).

Fixes #1074

### CODE REVIEW

File: en/deploy/signup_pythonanywhere.md [hunk 1]
```diff
@@ -1,11 +1,15 @@
+PythonAnywhere is a service for running Python code on servers "in the cloud". We'll use
+it for hosting our site, live and on the Internet.
 
-Sign up for a free "Beginner" account on PythonAnywhere:
+Sign up for a "Beginner" account on PythonAnywhere (the free tier is fine, you don't need a credit card).
 
 * [www.pythonanywhere.com](https://www.pythonanywhere.com/)
 
+<img src="images/pythonanywhere_beginner_account_button.png" alt="The PythonAnywher signup page showing button to create a the free 'Beginner' account" />
```
Comment (ekohl) [inline en/deploy/signup_pythonanywhere.md]: Typo: you missed an e in PythonAnywhere.

File: en/deploy/signup_pythonanywhere.md [hunk 2]
```diff
@@ -1,11 +1,15 @@
...
 
 > **Note** When choosing your username here, bear in mind that your blog's URL will take the form `yourusername.pythonanywhere.com`, so choose either your own nickname or a name for what your blog is all about.
 
+
```
Comment (ekohl) [inline en/deploy/signup_pythonanywhere.md]: This may be a nitpick, but why add this newline?

File: en/deploy/signup_pythonanywhere.md (hunk 1 above)
Comment (jhodgdon-drp) [reply en/deploy/signup_pythonanywhere.md]: I also thought we weren't supposed to use IMG tags, but instead the ! notation?

**Comment by hjwp** [comment]
think i've addressed those two, all good?

**Comment by The-Compiler** [comment]
The image seems to be broken on https://tutorial.djangogirls.org/en/installation/ - I opened #1356 in the hopes of fixing that.

**Pull request merged**